From d703905bda9f0c7461a47e0d5ff8d30c40b155a8 Mon Sep 17 00:00:00 2001
From: "xiaoya.lin" <xiaoya.lin@amlogic.com>
Date: Thu, 25 May 2023 16:18:35 +0800
Subject: [PATCH 3/3] jpegenc: CB2 modify height pitch. [1/1]

PD#SWPL-120819

Problem:
need modify height pitch.

Solution:
modify height pitch.

Verify:
local

Change-Id: Iffdac21f9cf9d8f6c0fb2c91f70e22335a61dcbe
Signed-off-by: xiaoya.lin <xiaoya.lin@amlogic.com>
---
 drivers/frame_sink/encoder/jpeg/jpegenc.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/frame_sink/encoder/jpeg/jpegenc.c b/drivers/frame_sink/encoder/jpeg/jpegenc.c
index 6d44e1762..02f46c9b0 100644
--- a/drivers/frame_sink/encoder/jpeg/jpegenc.c
+++ b/drivers/frame_sink/encoder/jpeg/jpegenc.c
@@ -2907,7 +2907,8 @@ static s32 set_jpeg_input_format(struct jpegenc_wq_s *wq,
             input = wq->InputBuffStart;
 
         picsize_x = ((cmd->encoder_width + 15) >> 4) << 4;
-        picsize_y = ((cmd->encoder_height + 15) >> 4) << 4;
+        //picsize_y = ((cmd->encoder_height + 15) >> 4) << 4;
+        picsize_y = cmd->encoder_height;
 
         if (get_cpu_major_id() > AM_MESON_CPU_MAJOR_ID_SC2) {
             /*
-- 
2.25.1

